docs/src/examples/components/Alert/Variations/AlertExampleAttached.shorthand.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import { Alert, Input } from '@stardust-ui/react'
+
+const AlertExampleShorthand = () => (
+  <>
+    <Alert attached content="This is a top attached alert" />
+    <Input fluid placeholder="Name..." />
+    <br /> <br />
+    <Input fluid placeholder="Surname..." />
+    <Alert attached="bottom" content="This is a bottom attached alert" />
+  </>
```

Comment:
Can we use `Flex` with `gap` there?

Reply:
we can but code will double in size for the same effect:
```jsx
const AlertExampleShorthand = () => (
  <Flex gap="gap.smaller" column>
    <Flex.Item>
      <>
        <Alert attached content="This is a top attached alert" />
        <Input fluid placeholder="Name..." />
      </>
    </Flex.Item>
    <Flex.Item>
      <>
        <Input fluid placeholder="Surname..." />
        <Alert attached="bottom" content="This is a bottom attached alert" />
      </>
    </Flex.Item>
  </Flex>
)
```
=> 14 lines vs 7 lines currently:
```jsx
const AlertExampleShorthand = () => (
  <>
    <Alert attached content="This is a top attached alert" />
    <Input fluid placeholder="Name..." />
    <br /> <br />
    <Input fluid placeholder="Surname..." />
    <Alert attached="bottom" content="This is a bottom attached alert" />
  </>
)
```
still wanna do it?

Reply:
Valid point, let's keep it as is